src/inventory_ui.cpp

```diff
@@ -2468,6 +2468,9 @@ drop_locations inventory_drop_selector::execute()
     int count = 0;
     while( true ) {
         ui_manager::redraw();
+        if( get_option<std::string>( "INVENTORY_HIGHLIGHT" ) != "disable" ) {
+            highlight();
```

Comment:
`highlight()` is also calling `ui_manager::redraw()`, so the latter is called twice every redraw unnecessarily. Please remove `ui_manager::redraw()` from `highlight` and call `highlight` before the `ui_manager::redraw()` calls on line 2470/2123.